client/gutenberg/extensions/publicize/store/selectors.js

```diff
@@ -6,5 +6,18 @@
  * @return {Array} List of connections.
  */
 export function getFailedConnections( state ) {
-	return state.filter( connection => ! connection.test_success );
+	return state.filter( connection => false === connection.test_success );
+}
+
+/**
+ * Returns Publicize connections that require reauthentication from users. For example, when LinkedIn switched its API from v1 to v2.
```

Comment:
It would be useful to list all cases when this can happen here, or at least a reference to where we can find more information about the cases this can happen.